PointOfInteraction/SmartPointOfInteraction/doc/spec.md

```diff
@@ -0,0 +1,69 @@
+# SmartPointOfInteraction
+
+## Description
+
+A Smart Point of Interaction defines a place with technology to interact with users, for example, through Beacon technology from Apple, Eddystone/Physical-Web from Google or other proximity-based interfaces. Since the interactive area could be composed by more than one device providing the technology, this model encompasses a group of SmartSpot devices.
+
+The data model includes information regarding the area/surface covered by the technology (i.e., the area covered by Bluetooth Low Energy-based Beacon), a way to specify the functionality intervals (i.e. when interactive points are available) and the link to the multimedia resource where users will interact (i.e. Web Apps, etc.). Additionally, the data model may reference to another NGSI entity such as a Parking, a Point of Interest (POI), etc. with enriched interaction provided by this Smart Point of Interest.
+
+
+This entity is purely virtual, is not a device mapping.
+
+## Data Model
+
++ `id` : Unique identifier. 
+
++ `type` : Entity type. It must be equal to `SmartPointOfInteraction`.
+
++ `category` : Defines the type of interaction.
+    + Attribute type: List of [Text](http://schema.org/Text)
+    + Allowed values: `information`, `entertainment`, `infotainment`, `co-creation` or any other extended value defined by the application.
+    + Mandatory
+    
++ `areaCovered` : Defines the area covered by the Smart Point of Interaction using geoJSON format.
```

Comment:
it would be good to indicate what kind of geometrical feature can be used to represent this property